main.go

```diff
@@ -346,6 +396,20 @@ func cursorRight(g *gocui.Gui, v *gocui.View) error {
 			return err
 		}
 	}
+	return refreshDetailsView(g)
+}
+
+func refreshDetailsView(g *gocui.Gui) error {
+	mainView, _ := g.View("main")
+	_, cy := mainView.Cursor()
+
+	detailsView, _ := g.View("details")
+	detailsView.Clear()
+
+	repo := reposPerLang[lang][cy]
+
+	fmt.Fprintf(detailsView, "%s %s\n", repo.language, repo.stars)
```

Comment:
How do you feel add `☆` before show stars?
(I think it's probably a problem to get used to, but I didn't know any numbers at first.)